File: WebSpeak v.2/VitaliiMahlona/DAL/Models/DTOStatistics.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Models
{
    public class DTOStatistics
    {
        public List<DTOTestResults> testResults { get; set; }
        public List<DTO> LangList { get; set; }
        public List<DTO> CatList { get; set; }
        public List<DTO> SubCatList { get; set; }
        public List<DTOTotalRating> TotalRatings { get; set; }
        public List<DTOTotalRating> LangRatings { get; set; }
    }
}

File: WebSpeak v.2/VitaliiMahlona/LearningLanguagesAngular/ClientApp/app/models/DTOStatistics.ts
import { DTO } from './DTO'
import { DTOTestResults } from './DTOTestResults' 
import { DTOTotalRating } from './DTOTotalRating'

export class DTOStatistics {
    public testResults: DTOTestResults[];
    public langList: DTO[];
    public catList: DTO[];
    public subCatList: DTO[];
    public totalRatings: DTOTotalRating[];
    public langRating: DTOTotalRating[];
}
